packages/@aws-cdk/cloudformation-include/test/invalid-templates.test.ts

```diff
@@ -130,7 +128,7 @@ describe('CDK Include', () => {
     includeTestTemplate(stack, 'alphabetical-string-passed-to-number.json');
 
     expect(() => {
-      SynthUtils.synthesize(stack);
+      core.App.of(stack)?.synth();
     }).toThrow(/"abc" should be a number/);
   });
 
```

Comment:
The lack of `SynthUtils` is really biting us huh 😕.

How about we add an `App` variable next to `stack`, initialize in the `beforeEach()`, and then just do `app.synth()` instead of this `App.of()?.synth()` dance?